### ISSUE TITLE
При активации опции показа на папке 4 сайтов, ломается отображение папки в интерфейсе

### ISSUE THREAD

**Issue by tipodanet** [k-ivan/visual-bookmarks-chrome#186]
хотя бы четкими границами. Какой-нить контрастный (светлой/темной теме) border 1рх 
![image](https://github.com/user-attachments/assets/299b2dff-0c2a-4c00-a5fe-50336a091bab)

Это папка, вероятно,  с именем "Tracking"
Рядом с ней была такая же "черная дыра" с именем "Cashback". Пока я думал, что это за TIPS на пустом месте такая странная,  я ее удалил. Сколько там было внутри закладок, не известно. Хорошо ,что "Cashback" с их обманами и разводами меня меньше всего интересует :).

**Comment by k-ivan** [comment]
Честно говоря, ничего не понял, что значит обозначить папку, почему она у вас прозрачная на скрине? Опишите подробнее, как так получилось?

**Comment by tipodanet** [comment]
сейчас у меня стилей никаких нет. И те что могут быть, только по анимации. Цвета не трогаю. Так выглядят ВЗ в EDGE. Это папки без миниатюр.

**Comment by k-ivan** [comment]
Очевидно что есть, так как папки отображаются, проверил в ФФ, Хроме и эдже, везде все ок.

**Comment by tipodanet** [comment]
Честно, я не в курсе
![image](https://github.com/user-attachments/assets/cde78d57-549e-471b-9301-0747a8843523)
Расширение DARK READER отключено. Остальное не по стилям. Все пробелы -- это папки.

**Comment by tipodanet** [comment]
я так понимаю, что EDGE  при установке спрашивает про Импорт, и берет закладки из аккаунта Chome. Сейчас в меня даже нет этого Chome. Я не могу посмотреть эти закладки в оригинале. 
А что я там увижу ? Папки все одинаковые, имена разные.

**Comment by k-ivan** [comment]
Очевидно какая-то локальная проблема, с самим расширением проблем нет, папки отображается во всех поддерживаемых браузерах. Не могу воспроизвести

**Comment by tipodanet** [comment]
Создание всех миниатюр внутри исправляет ситуацию . Но хотелось бы и пустые такие же

**Comment by k-ivan** [comment]
Повторюсь, это какой-то локальный кейс, даже не представляю, что нужно сделать, чтобы воспроизвести.

**Comment by k-ivan** [comment]
Попробуйте убрать все ваши стили

**Comment by tipodanet** [comment]
![image](https://github.com/user-attachments/assets/a302dd3c-72da-4aed-b684-988ffa0999ba)
расскажите, что значит 3 кнопка сверху ?

**Comment by k-ivan** [comment]
если на вкладке "Общие" включена синхронизация, то все настройки будут синхронизироваться с облаком браузера(если Вы авторизовались в браузере). То есть, установив расширение на другом устройстве, и активировав синхронизацию, все настройки подтянуться из облака(если ранее были туда синхронизированы с одного из устройств). Эта кнопка очищает данные настроек в облаке. То есть другое устройство не сможет подхватить(при включенной опции) настройки с другого устройства.

**Comment by tipodanet** [comment]
Понятно, Настройки можно восстановить, это не "фатальная" кнопка. Она не удаляет сами Закладки ?
Полагаю, что нет. Эту кнопку надо уточнить описание. "Данные" -- звучит, как "все". Речь только о настройках Расширения ?

**Comment by tipodanet** [comment]
Почему я такой испуганный ?  В Вивальди тоже есть кнопка СБРОС. Она удаляет ВСЕ, без возможности восстановления или отката. Обратная закачка только из самого браузера, что остался у юзера. Ужи были стоны "я нажал, не подумал, вы не предупредили, не объяснили , а где Ctrl+Z  ... ALL MY LIFE (((( "
поэтому я уточняю 2 раза.

**Comment by k-ivan** [comment]
Сверьте настройки до синхронизации и после(можно скринами)

- очистите **локальные ** настройки на дефолт
- проверьте, что все отображается и сделайте скрины всех настроек
- затем включите синхронизацию(он предложит загрузить уже имеющиеся настройки с облака), соглашайтесь, затем делайте скрины настроек, чтобы сравнить, какие настройки были и какие стали

**Comment by tipodanet** [comment]
да, после "Сброс настроек с сервера" и всех кнопок сброса, ситуация исправлена.  
Спасибо, все работает

**Comment by k-ivan** [comment]
Нужно было сравнить, какая настройка приводила к этому, сбросив локальные настройки, я писал выше. Но теперь уже поздно.

**Comment by tipodanet** [comment]
именно так я и делал. 2 раза сбрасывал локальные. Но не трогал "сбросить ДАННЫЕ", как самая страшная кнопка. 
Все было нормально, до ВКЛ  "Сброс настроек с сервера". Она была последняя. После нее GHOST.
Ну понятно, надо было протестить все настройки без сброса.  Если встретится,  отправлю депешу.
Это повод для следующего предложения --

**Comment by k-ivan** [comment]
![Image](https://github.com/user-attachments/assets/21272a5f-edc2-4546-8a15-962276b4e8d9)

Случайно обнаружил, действительно, есть проблема, связана она с активацией опции "Показывать 4 случайных сайта у папки"
